py/selenium/webdriver/firefox/webdriver.py

```diff
@@ -17,11 +17,12 @@
 # specific language governing permissions and limitations
 # under the License.
 
-
+http_exception = None
 try:
     import http.client as http_client
 except ImportError:
     import httplib as http_client
+    http_exception = http_client.HTTPException
```

Comment:
i'm not a python-ista (?), but wouldn't this be better:

 ```python
http_exception = None
try:
    import http.client as http_client
except ImportError:
    import httplib as http_client

http_exception = http_client.HTTPException
```

that way it assigns http_exception either case

Reply:
Fix and restore to the original format and release old documents